docs/azure_jumpstart_ag/contoso_supermarket/troubleshooting/_index.md

```diff
@@ -4,4 +4,41 @@ weight: 100
 toc_hide: true
 ---
 
-# Overview
\ No newline at end of file
+# Jumpstart Agora - Contoso Supermarket troubleshooting
+
+## Basic Troubleshooting
+
+Occasionally deployments of the Agora Contoso Supermarket experience may fail at various stages. Common reasons for failed deployments include:
+
+- Invalid service principal id, service principal secret or service principal Azure tenant ID provided in _azuredeploy.parameters.json_ file.
+- Invalid SSH public key provided in _azuredeploy.parameters.json_ file.
+  - An example SSH public key is shown here. Note that the public key includes "ssh-rsa" at the beginning. The entire value should be included in your _main.parameters.json_ file.
+
+      ![Screenshot showing SSH public key example](./ssh_example.png)
+
+- User has not forked the [Jumpstart-agora-apps repository](https://github.com/microsoft/jumpstart-agora-apps). To simulate the developer experience, you must first fork the sample apps repo so that you have your own version of the underlying source code to work with. Instructions on how to fork this repo are included in the [deployment guide](https://github.com/microsoft/azure_arc/blob/jumpstart_ag/docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md).
```

Comment:
```suggestion
- User has not forked the [jumpstart-agora-apps GitHub repository](https://github.com/microsoft/jumpstart-agora-apps). To simulate the developer experience, you must first fork the sample apps repo so that you have your own version of the underlying source code to work with. Instructions on how to fork this repo are included in the [deployment guide](https://github.com/microsoft/azure_arc/blob/jumpstart_ag/docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md).
```